wordpress.org/public_html/wp-content/plugins/wporg-gp-virtual-projects/wporg-gp-virtual-projects.php

```diff
@@ -0,0 +1,233 @@
+<?php
+/**
+ * Plugin Name: WPORG Virtual projects for patterns.
+ * Description: Creates virtual projects for the patterns.
+ * Author: the WordPress team.
+ */
+class WPorg_Virtual_Projects {
+
+	/**
+	 * Adds filters to subprojects and to project status.
+	 */
+	public function __construct() {
+		add_filter( 'gp_subprojects', array( $this, 'add_virtual_projects' ), 10, 1 );
+		add_filter( 'gp_get_project_status', array( $this, 'gp_get_project_status' ), 10, 4 );
+		add_filter( 'gp_translation_table_after', array( $this, 'add_pattern_preview' ), 10, 2 );
```

Comment:
```suggestion
		add_filter( 'gp_translation_table_classes', array( $this, 'gp_translation_table_classes' ) );
		add_action( 'gp_translation_table_after', array( $this, 'add_pattern_preview' ), 10, 2 );
```

See https://github.com/GlotPress/GlotPress/pull/1665/files#r1279315477